src/components/data_display/helpers.ts

```diff
@@ -306,7 +300,8 @@ export const constructDetailsResponse = async (
   const evaluationFieldArray: string[] = []
   const item = columnDefinitionMasterList.reduce(
     (obj: { [key: string]: any }, { columnName, isBasicField, columnDefinition }) => {
-      if (isBasicField) obj[columnName] = fetchedRecord[columnName]
+      if (isBasicField && !columnDefinition?.valueExpression)
+        obj[columnName] = fetchedRecord[columnName]
```

Comment:
This is all that's needed to fix #829. This just means we can have a `valueExpression` without needing to change the name of the column to a custom name.